translations/mod-fqm-manager/en.json

```diff
@@ -787,6 +788,7 @@
   "entityType.simple_item_details.permanent_location_id": "Permanent location ID",
   "entityType.simple_item_details.purchase_order_line_identifier": "Purchase order line identifier",
   "entityType.simple_item_details.statistical_code_ids": "Statistical code IDs",
+  "entityType.simple_item_details.statistical_codes": "Statistical codes",
```

Comment:
diambiguate
```suggestion
  "entityType.simple_item_details.statistical_codes": "Statistical code names",
```

Reply:
Counter-point:

(from the naming conventions in `translations/`)
> - Do not refer to arrays as 'lists' directly; instead, use plurals
>   - _Example_: `Statistical codes` instead of `Statistical code list`